src/wp-includes/class-wp-theme-json.php

```diff
@@ -1624,13 +1681,16 @@ public function get_svg_filters( $origins ) {
 	 * Returns whether a presets should be overridden or not.
 	 *
 	 * @since 5.9.0
+	 * @deprecated 6.0.0 Use {@see 'get_metadata_boolean'} instead.
 	 *
 	 * @param array      $theme_json The theme.json like structure to inspect.
 	 * @param array      $path Path to inspect.
 	 * @param bool|array $override Data to compute whether to override the preset.
 	 * @return boolean
 	 */
 	protected static function should_override_preset( $theme_json, $path, $override ) {
+		_deprecated_function( __METHOD__, '6.0.0' );
```

Comment:
We should specify the replacement in the third parameter.

Reply:
Done in 15526a2af7